benches/payout-stakers/src/main.rs

```diff
@@ -30,23 +30,25 @@ const NOMINATE_CALL_BATCH_LIMIT: usize = 128;
 
 fn main() -> Result<(), anyhow::Error> {
     let address = "127.0.0.1:9944";
+    let ssl = false;
     let sudoer = AccountKeyring::Alice.pair();
 
     env_logger::init();
     info!("Starting benchmark with config ");
 
-    let connection = create_connection(address).set_signer(sudoer);
+    let connection = create_connection(address, ssl).set_signer(sudoer);
 
     let accounts = generate_1024_accounts(&connection);
-    let validators = set_validators(address);
+    let validators = set_validators(address, ssl);
```

Comment:
Ideally, we should not be passing `ssl` here but move to refactor `set_validator` API to have `Connection&` as an argument. I know it's more work as we'd need to update all depndess of `set_validator`, but then we don't need to pass through all crates this bool flag. Even more ideally we should move `set_validators` to `aleph-client`, please see other functions there as we're trying to establish some kind of convention that all `aleph-client` functions should have `Connection&`.

Reply:
done